src/view/invoice.js

```diff
@@ -38,13 +46,22 @@ const InvoiceView = ({ store, nav, invoice }) => (
     <MainContent>
       <Card>
         <BalanceLabel style={styles.balance}>
+          <BalanceLabelNumeral
+            style={[
+              styles.fiatUnit,
+              { fontSize: store.settings.displayFiat ? font.sizeXXXL : 0 },
+            ]}
+          >
+            $
+          </BalanceLabelNumeral>
           <AmountInputField
             autoFocus={true}
-            value={store.invoice.amount}
```

Comment:
All our TextInputs are currently controlled react components. We need to add this again.